tests/unit/test_hybrid_layer.py

```diff
@@ -60,7 +60,7 @@ def test_initialization_different_encoders(self, cohere_encoder, openai_encoder)
     def test_add_route(self, openai_encoder):
         route_layer = HybridRouteLayer(encoder=openai_encoder)
         route = Route(name="Route 3", utterances=["Yes", "No"])
-        route_layer.add(route)
+        route_layer._add_routes([route])
```

Comment:
I know it makes no effective difference here, but should it not point to `.add` as we are testing the endpoints users will be testing? (Thinking forward in case `.add` changes in any way beyond just calling `._add_routes`